### ISSUE TITLE
WCAG link purpose

### ISSUE THREAD

**Issue by fgeierst** [fgeierst/bolivianisches-kinderhilfswerk-stiftung#14]
The links to the blog detail pages fail [WCAG 2.4.4 Link purpose](https://ergebnis.bitvtest.de/pruefschritt/wcag-21-web/wcag-21-web-2-4-4-aussagekraeftige-linktexte).

Example: https://bolivianisches-kinderhilfswerk-stiftung.de/blog/
